### PR TITLE
Remove unsed variables

### PR DESCRIPTION
**Pull request by jayvdb**

### CODE REVIEW

File: piksi_tools/console/update_downloader.py [hunk 1]
```diff
@@ -31,7 +31,7 @@ def __init__(self, root_dir=swift_path):
             f.raise_for_status()
             self.root_dir = root_dir
             f.close()
-        except (requests.ConnectionError, json.decoder.JSONDecodeError) as ce:
+        except (requests.ConnectionError, json.decoder.JSONDecodeError):
             self.index = None
             raise RuntimeError("Unable to download index from {0}.".format(INDEX_URL))
```
Comment (jayvdb) [inline piksi_tools/console/update_downloader.py:36]: we could append `{ce}` here

File: piksi_tools/console/update_view.py [hunk 2]
```diff
@@ -90,7 +90,7 @@ def _check_thd(self):
                 socket.create_connection(
                     NetworkCheck.SERVER, NetworkCheck.CONNECTION_TIMEOUT).close()
                 self._state = True
-            except Exception as ex:
```
Comment (jayvdb) [inline piksi_tools/console/update_view.py:93]: should this be logged?

**Review by silverjam — APPROVED** [review]

**Review by john-michaelburke — APPROVED** [review]
What tool are you using to catch all of these and do we have this set up in console_pp 😂

**Comment by jayvdb** [comment]
I use pyflakes directly.

**Review by notoriaga — APPROVED** [review]

**Pull request merged**